aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -1359,7 +1358,7 @@ Tensor scatter_reduce_two_cpu(const Tensor& self,
             out_cont_data[ind] *= value;
           } else if (reduce == "mean") {
             auto n = counts_data[ind];
-            out_cont_data[ind] = (out_cont_data[ind] * n + value) / (n + 1);
+            out_cont_data[ind] = out_cont_data[ind] + value;
```

Comment:
This method of computing mean is actually buggy for integer types, my bad. I updated it to follow pytorch_scatter which does sum and divides by a tensor that accumulated the 'n' for each index